web/source/keyboards/kmwkeyboards.ts

```diff
@@ -893,30 +887,51 @@ namespace com.keyman.keyboards {
       }
     };
 
+    /**
+     * Return a Promise of the merged keyboard stubs and error stubs
+     * @param {(KeyboardStub|ErrorStub)[]} keyboardStubs 
+     * @param {ErrorStub[]} errorStubs 
+     * @returns {Promise<KeyboardStub|ErrorStub)[]>}
+     */
+    private mergeStubs(keyboardStubs: (KeyboardStub|ErrorStub)[], errorStubs: ErrorStub[]) :
```

Comment:
Please choose a different name.  Perhaps something like `_stubReportHelper`, as it's used to help report the stubs found by keyboard requests?

There's already a `mergeStub()` function, and it does something **_very_** different.

Reply:
Propose a name more like `mergeAndResolveStubPromises`?